README.md

```diff
@@ -1 +1,39 @@
-# code-challenges-javascript
\ No newline at end of file
+# code-challenges-javascript
+This repo is for Javascript code-challenges
+## Set up
+1) cd into the directory you want this repository in - clone it down
+```
+git clone <path>
+```
+2) install jest for testing
+```
+npm install --save-dev jest
+```
+3) in package.json
+ ``` {
+  "scripts": {
+    "test": "jest"
+  }
+}
```

Comment:
What is this step?  If they cloned down the repo, the package.json is the one from the repo.